### PR TITLE
doc: update the architecture diagram of cluster-operator

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Update the architecture diagram to reflect the on-premise cluster lifecycle management capability of the cluster-operator, which was not present in the original architecture diagram

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Update the architecture diagram of cluster-operator
```

### CODE REVIEW

File: docs/content/en/docs/cluster-operator/image/on-premise.svg
Comment (hzxuzhonghu) [inline docs/content/en/docs/cluster-operator/image/on-premise.svg:1]: 图上面的元素跟其他的图不一致

Comment (hzxuzhonghu) [inline docs/content/en/docs/cluster-operator/image/on-premise.svg:1]: s/Passwordless/Password

Comment (hzxuzhonghu) [reply docs/content/en/docs/cluster-operator/image/on-premise.svg:1]: which component of cluster operator， it can be shown as well

Comment (Xieql) [reply docs/content/en/docs/cluster-operator/image/on-premise.svg:1]: yes, it is the old drawing, it need update

Comment (Xieql) [reply docs/content/en/docs/cluster-operator/image/on-premise.svg:1]: you are right

Comment (Xieql) [reply docs/content/en/docs/cluster-operator/image/on-premise.svg:1]: we can update the architecture diagram to reflect the on-premise cluster lifecycle management capability of the cluster-operator, which was not present in the original architecture diagram

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**